src/Routes/DetailPage/Components/DetailGeneral/DetailGeneral.tsx

```diff
@@ -261,7 +265,7 @@ export const DetailGeneral = (props: DetailGeneralProps) => {
           </Button>,
         ]}
       >
-        <TextInput value={editTitle} type="text" onChange={(value) => setEditTitle(value)} ouiaId="TextModalDomainTitle" />
+        <TextInput value={editTitle} type="text" onChange={(value) => setEditTitle(value)} ouiaId="TextModalDomainTitle" aria-label="New Title" />
```

Comment:
Q: what is `aria-label` for??

Reply:
It is responded on the PR description; I didn't realize.

Thanks for the enhancement; if I understand correctly it is used to make the page more accessible, for instance for text-speech for the browser.

Reply:
Yes. For more info you can check https://www.patternfly.org/accessibility/about-accessibility